src/test/tests/plots/molecule.py

```diff
@@ -0,0 +1,237 @@
+# ----------------------------------------------------------------------------
+#  CLASSES: nightly
+#
+#  Test Case:  molecule.py
+#
+#  Tests:      mesh      - 3D points
+#              plots     - Molecule
+#              operators - CreateBonds, Replicate
+#
+#  Programmer: Kathleen Biagas 
+#  Date:       June 15, 2021 
+#
+#  Modifications:
+#
+# ----------------------------------------------------------------------------
+
+def SetGradientBackground():
+    annot = GetAnnotationAttributes() 
+    annot.backgroundMode = annot.Gradient
+    annot.gradientBackgroundStyle = annot.Radial
+    annot.gradientColor1 = (102, 102, 153, 255)
+    annot.gradientColor2 = (0, 0, 0, 255)
+    annot.backgroundColor = (0, 0, 0, 255)
+    annot.foregroundColor = (255, 255, 255, 255)
+    SetAnnotationAttributes(annot)
+
+def SetWhiteBackground():
+    annot = GetAnnotationAttributes() 
+    annot.backgroundMode = annot.Solid
+    annot.foregroundColor = (0, 0, 0, 255)
+    annot.backgroundColor = (255, 255, 255, 255)
+    SetAnnotationAttributes(annot)
+
+def SaveTestImage(name):
+    # Save these images somewhat larger than a regular test case image
+    # to better see the molecules 
+    backup = GetSaveWindowAttributes()
+    swa = SaveWindowAttributes()
+    swa.width = 500
+    swa.height = 500
+    swa.screenCapture = 1
+    Test(name, swa)
+    SetSaveWindowAttributes(backup)
+
+
+def MoleculeOnly():
+    # images similar to those in Molecule Plot docs
```

Comment:
Just curious about something...is there any way we can have this code be a *quick recipe*? What I would attempt is to have the code here as you have but create a section in `quickrecipies.rst` that effectively *imports* the code block here into those docs like we do in `Expressions.rst` where we use reStructuredText `includeliteral` directive?

Reply:
Could you add a ticket if you want this please.

Reply:
Done. Added #5854